src/bci_build/templates.py

```diff
@@ -194,13 +194,24 @@
     """<services>
   <service mode="buildtime" name="{{ image.build_recipe_type }}_label_helper"/>
   <service mode="buildtime" name="kiwi_metainfo_helper"/>
+{%- set all_build_flavors = [""] %}
+{%-     if image.crate and image.build_flavor %}
+{%-         set all_build_flavors = image.crate.all_build_flavors(image) %}
+{%-     endif %}
+{%- for flavor in all_build_flavors %}
```

Comment:
yeah, I was trying to avoid storing the (essentially duplicated) information in the ContainerImage because it is only needed for this one case. Polluting yet another method into that ContainerImage seems rather messy to me. 

my intention was to move the generation of `_multibuild` and `_service` into the Crate entirely, but that was a bit too much for one commit so it'll come later on.